base/shell/explorer/taskswnd.cpp

```diff
@@ -1747,27 +2640,31 @@ class CTaskSwitchWnd :
 
         switch (nmh->code)
         {
-        case NM_CUSTOMDRAW:
```

Comment:
Not sure the reformatting here is useful.

Reply:
That's exactly to comply with the formatting rules